src/main/scala/mesosphere/marathon/core/storage/MarathonState.scala

```diff
@@ -0,0 +1,22 @@
+package mesosphere.marathon.core.storage
+
+import akka.util.ByteString
+import com.google.protobuf.{ MessageLite, Parser }
+import mesosphere.marathon.state.Timestamp
+
+trait MarathonState[Proto <: MessageLite] {
```

Comment:
I'll go ahead and rip this out too (and instead just implement the marshallers directly)